ProjFS.Mac/PrjFSKextTests/HandleOperationTests.mm

```diff
@@ -853,5 +1185,21 @@ - (void) testWriteDataToForkedPath {
     XCTAssertTrue(MockCalls::CallCount(ProviderMessaging_TrySendRequestAndWaitForResponse) == 0);
 }
 
+- (void) testIneligibleFilesystemType
+{
+    shared_ptr<mount> testMountNone = mount::Create("msdos", fsid_t{}, 0);
+    shared_ptr<vnode> testVnodeNone = vnode::Create(testMountNone, "/Volumes/USBSTICK");
+    XCTAssertEqual(
+        KAUTH_RESULT_DEFER,
+        HandleVnodeOperation(
```

Comment:
Should we also try calling `HandleFileOpOperation` for WILL_RENAME and validate that `s_pendingRenameCount` does not increase?

Reply:
Good idea; I'll add that to `HandleFileOpOperationTests(.mm)` though as it's not directly relevant to this test.